crm_schemas.go

```diff
@@ -0,0 +1,105 @@
+	Name               *HsStr  `json:"name"`
+	ToObjectTypeId     *HsStr  `json:"toObjectTypeId"`
+	UpdatedAt          *HsTime `json:"updatedAt"`
+}
+
+type CrmSchemaLabels struct {
+	Plural   *HsStr `json:"plural"`
+	Singular *HsStr `json:"singular"`
+}
+
+type CrmSchemasList struct {
+	Results []*CrmSchema
+}
+
+type CrmSchema struct {
+	Archived                   *HsBool                 `json:"archived,omitempty"`
+	Associations               []*CrmSchemaAssociation `json:"associations,omitempty"`
+	FullyQualifiedName         *HsStr                  `json:"fullyQualifiedName,omitempty"`
+	ID                         *HsStr                  `json:"id,omitempty"`
+	Labels                     *CrmSchemaLabels        `json:"labels,omitempty"`
+	MetaType                   *HsStr                  `json:"metaType,omitempty"`
+	Name                       *HsStr                  `json:"name,omitempty"`
+	ObjectTypeId               *HsStr                  `json:"objectTypeId,omitempty"`
+	PrimaryDisplayProperty     *HsStr                  `json:"primaryDisplayProperty,omitempty"`
+	Properties                 []*CrmProperty          `json:"properties,omitempty"`
+	RequiredProperties         []*HsStr                `json:"requiredProperties,omitempty"`
+	Restorable                 *HsBool                 `json:"restorable,omitempty"`
+	SearchableProperties       []*HsStr                `json:"searchableProperties,omitempty"`
+	SecondaryDisplayProperties []*HsStr                `json:"secondaryDisplayProperties,omitempty"`
+	CreatedAt                  *HsTime                 `json:"createdAt,omitempty"`
+	UpdatedAt                  *HsTime                 `json:"updatedAt,omitempty"`
+}
+
+// CrmSchemasService is an interface of CRM schemas endpoints of the HubSpot API.
+// Reference: https://developers.hubspot.com/docs/api/crm/crm-custom-objects
+type CrmSchemasService interface {
+	List() (*CrmSchemasList, error)
+	Create(reqData interface{}) (*CrmSchema, error)
+	Get(schemaIdentifier string) (*CrmSchema, error)
+	Delete(schemaIdentifier string, option *RequestQueryOption) error
+	Update(schemaIdentifier string, reqData interface{}) (*CrmSchema, error)
+}
+
+// CrmSchemasServiceOp handles communication with the CRM schemas endpoint.
+type CrmSchemasServiceOp struct {
+	client         *Client
+	crmSchemasPath string
+}
+
+var _ CrmSchemasService = (*CrmSchemasServiceOp)(nil)
+
+func (s *CrmSchemasServiceOp) List() (*CrmSchemasList, error) {
+	var resource CrmSchemasList
+	if err := s.client.Get(s.crmSchemasPath, &resource, nil); err != nil {
+		return nil, err
+	}
+	return &resource, nil
+}
+
+func (s *CrmSchemasServiceOp) Create(reqData interface{}) (*CrmSchema, error) {
+	var resource CrmSchema
+	if err := s.client.Post(s.crmSchemasPath, reqData, &resource); err != nil {
+		return nil, err
+	}
+	return &resource, nil
+}
+
+func (s *CrmSchemasServiceOp) Get(schemaIdentifier string) (*CrmSchema, error) {
```

Comment:
I'm going to follow the documentation and use objectType, is that a problem?

```suggestion
func (s *CrmSchemasServiceOp) Get(objectType string) (*CrmSchema, error) {
```